website/screens/common/QuickNavContainer.tsx

```diff
@@ -77,7 +77,8 @@ const Container = styled.div`
 const ContentContainer = styled.div`
   display: flex;
   flex-direction: column;
-  width: 800px;
+  width: 100%;
+  max-width: 800px;
```

Comment:
The fix works fine but the table becomes difficult to read due to the amount of text and small space for the description column.
Since we already have the scroll making the column wider will help with readability.